pkgs/flutter_http_example/lib/main.dart

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
```

Comment:
```suggestion
// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
```

Reply:
This is copied from pkgs/cupertino_http so I think that this copyright is correct.